src/component/BooksRender.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RemoveBooks from './Removebook';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCoffee } from '@fortawesome/free-solid-svg-icons';
+
+const Books = ({ id, title, author }) => (
+  <div className="bookSite">
+    <div className="info">
+      <div className="left-section">
+        <ul className="align">
+          <li className="genre books">Action</li>
+          <div>
+            <li className="booktitle books">{title}</li>
+          </div>
+          <li className="bookauthor books">{author}</li>
+        </ul>
+
+        <div className="book-actions">
+          <button className="book-btn" type="button">
+            Comments
+          </button>
+          <span>|</span>
+          <RemoveBooks id={id} />
+          <span>|</span>
+          <button className="book-btn" type="button">
+            Edit
+          </button>
+        </div>
+      </div>
+
+      <div className="center-section">
+        <div className="oval">
+          <CircularProgressbar
+            value={32}
+            text={`${32}%`}
+            styles={{
+              path: {
+                stroke: '#fff',
+              },
+              text: {
+                fill: '#fff',
+                fontSize: '14px',
+              },
+            }}
```

Comment:
- [ ] Good job on making changes as requested by the previous reviewer, however, I would like to inform you that I have experienced some errors while trying to run your project about modules not being found. As a result, I could not evaluate if the requested changes have been implemented correctly.  Let me show you the error screens
![image](https://user-images.githubusercontent.com/90222110/204104543-bf1c4ebb-305f-4524-8a45-86bdb4bba8d1.png)

![image](https://user-images.githubusercontent.com/90222110/204104550-fb235a9e-44b2-4d0c-ab42-0ddc9067ed41.png)


I think this is because you are using the packages you have not installed. I have checked your `package.json file` and I can't find them. Please add them 😃 👍 
![image](https://user-images.githubusercontent.com/90222110/204104612-913495f3-7e8e-42c7-a2c6-15ad970c794c.png)

Reply:
Hi @BertrandConxy it is because I installed and am using react-circuler-progress-bar and font awesome for react. It works perfectly for me I don't know what I am supposed to do

Reply:
@BertrandConxy I guess i will send it for another review but I only have one left

Reply:
@BertrandConxy Should I resend it? or should I do something special before I do? Because react-circuler-progress-bar and font awesome for react are important for the project